R/log.R

```diff
@@ -285,8 +288,39 @@ log_write <- function(file = NA,
                         write_log_element("log_path", "Log path: "))
 
    writeLines(cleaned_log_vec,
-              con = file.path(get_log_element("log_path"),
-                              get_log_element("log_name")))
+                 con = file.path(get_log_element("log_path"),
+                                 get_log_element("log_name")))
+   if (include_rds){
+      rds_fields <- c(
+         "end_time", "start_time", "run_time", "user", "hash_sum",
+         "log_path", "log_name", "file_path", "file_name",
+         "unapproved_packages_functions", "errors", "warnings"
+      )
+      log_options <- as.list(getOption('log.rx'))
+      cleaned_log_list <- purrr::map2(
+         log_options,
+         names(log_options),
+         function(i, x){
+            if(x %in% c("messages", "output", "result")){
+               if(x %in% to_report){
+                  return(i)
+               }
+            } else if(x %in% c(names(log_cleanup()), rds_fields)){
+               return(i)
+            }
+         }
+      )
+      cleaned_log_list$session_info <- session_info(info = "all")
```

Comment:
I think you have to do this because we capture the output of session_info and store it, rather than just storing the session info object.  I think we can move the capture output to the write function so this will still print pretty, and then you won't need to get the session info again.